packages/react-relay/modern/__tests__/ReactRelayPaginationContainer-test.js

```diff
@@ -656,8 +656,9 @@ describe('ReactRelayPaginationContainer', () => {
     });
 
     it('updates after pagination (if more results)', () => {
```

Comment:
Can you make this an `async () =>`

And then use `await environment.mock.resolve(...`?